examples/edit.cpp

```diff
@@ -0,0 +1,22 @@
+#include <iostream>
+#include <cstdlib>
+#include <string>
+
+#include "vega/dictionary/dictionary.h"
+#include "vega/dicom/file.h"
+
+int main(int argc, char *argv[]) {
+  if (argc < 3){
+    std::cout<<"Usage: edit input_file output_file"<<std::endl;
+    return 1;
+  }
+  // Optional you can set the dictionary file
+  // vega::dictionary::Dictionary::set_dictionary("/path/to/dictionary/dictionary.txt");
+  vega::dicom::File file(argv[1]);
+
+  std::string& name = file.data_set()->element<vega::dictionary::PatientName>()->manipulator()->at(0);
+  name = "Smith^Alice";
```

Comment:
I like the idea of having an example which shows how to edit a file. Could you update this logic though to do some bounds checking? The `element<T>` method for instance could return a `nullptr` if there is no patient name, and the manipulator might not have a 0th element yet (the manipulator acts as a `std::vector<std::string>`, so you can either check that `size() >= 1`, or just loop through all the names and change them, maybe something like this:

```c++
auto element = file.data_set->element<vega::dictionary::PatientName>();
if (element) {
  // If there are patient names, set them all to "Smith^Alice"
  for (auto& name : element->manipulator()) {
    name = "Smith^Alice";
  }
} else {
  // If there are no patient names, add one with value "Smith^Alice"
  element = file.data_set()->new_element<vega::dictionary::PatientName>();
  element->manipulator()->push_back("Smith^Alice");
}
```

If you use this, please test this locally though in case I made any minor errors.